README.md

```diff
@@ -187,65 +298,118 @@ Translations in Django templates use one of two template tags, `{% t %}` and
+        other {{host} invites {guest} and # other people to her party.}
+      }
+    }
+    male {
+      {total_guests, plural, offset:1
+        =0 {{host} does not give a party.}
+        =1 {{host} invites {guest} to his party.}
+        =2 {{host} invites {guest} and one other person to his party.}
+        other {{host} invites {guest} and # other people to his party.}
+      }
+    }
+    other {
+      {total_guests, plural, offset:1
+        =0 {{host} does not give a party.}
+        =1 {{host} invites {guest} to their party.}
+        =2 {{host} invites {guest} and one other person to their party.}
+        other {{host} invites {guest} and # other people to their party.}
+      }
+    }
+  }""",
+  gender_of_host="female",
+  total_guests=current_event.total_guests,
+  host=current_event.host.user.name,
+  guest=guest.name,
+)
+```
+
+##### Inline and Block Syntax
 
 Both template tags support two styles:
 
 1. The inline syntax
 
-   ```
-   {% t  <source>[|filters...] [key=param[|filters...]...] [as <var_name>] %}
-   {% ut <source>[|filters...] [key=param[|filters...]...] [as <var_name>] %}
-   ```
+```python
+{% t <source>[|filters...] [key=param[|filters...]...] [as <var_name>] %}
+{% ut <source>[|filters...] [key=param[|filters...]...] [as <var_name>] %}
+```
 
 2. The block syntax
 
-   ```
-   {% t  [|filters...] [key=param[|filters...]...] [as <var_name>] %}
-     <source>
-   {% endt %}
-   {% ut [|filters...] [key=param[|filters...]...] [as <var_name>] %}
-     <source>
-   {% endut %}
-   ```
+```python
+{% t [|filters...] [key=param[|filters...]...] [as <var_name>] %}
+<source>
+{% endt %}
 
-   In general, the outcome of the block syntax will be identical to the inline
-   syntax, if you had supplied the block contents as a literal argument. ie
-   this:
+{% ut [|filters...] [key=param[|filters...]...] [as <var_name>] %}
+<source>
+{% endut %}
+```
 
-   ```
-   {% t ... %}hello world{% endt %}
-   ```
+In general, the outcome of the block syntax will be identical to the inline
+syntax, if you had supplied the block contents as a literal argument. ie
+this:
 
-   should be identical to this:
+```python
+{% t ... %}hello world{% endt %}
+```
 
-   ```
-   {% t "hello world" ... %}
-   ```
+should be identical to this:
+
+```python
+{% t "hello world" ... %}
+```
 
-   With the block syntax, however, you can include characters that would be
-   problematic with the inline syntax, like quotes (`"`) or newlines.
+With the block syntax, however, you can include characters that would be
+problematic with the inline syntax, like quotes (`"`) or newlines.
 
-##### Plurals and other complex structures
+##### Plurals and Other Complex Structures
 
 The Transifex Toolkit supports the [ICU Message Format](http://userguide.icu-project.org/formatparse/messages).
 
 Using the Message Format syntax you can support various types of logic, with
 the same template tag:
 
-```
+```python
```

Comment:
The examples aren't python, but django template code. I thought github markdown didn't support highlighting for django templates so I left them without highlighting. But I was wrong:

You can do it with ` ```django`

```django
{% load transifex %}

{% t "hello world" var=var|safe %}
```

https://docs.djangoproject.com/en/3.0/topics/i18n/translation/#how-django-discovers-language-preference

Reply:
Done